tiledb/sm/array_schema/domain.h

```diff
@@ -224,6 +224,47 @@ class Domain {
   template <class T>
   int cell_order_cmp(const T* coords_a, const T* coords_b) const;
 
+  /**
+   * Checks the cell order of the input coordinates on the given dimension.
+   *
+   * @param The dimension to compare on.
+   * @param coord_buffs The input coordinates, given in separate buffers,
+   *     one per dimension. The buffers are sorted in the same order of the
+   *     dimensions as defined in the array schema.
+   * @param a The position of the first coordinate tuple across all buffers.
+   * @param b The position of the second coordinate tuple across all buffers.
+   * @param d The dimension index to compare on.
+   * @return One of the following:
+   *    - -1 if the first coordinates precede the second on the cell order
+   *    -  0 if the two coordinates have the same cell order
+   *    - +1 if the first coordinates succeed the second on the cell order
+   */
+  template <class T>
+  static int cell_order_cmp(
```

Comment:
What's the motivation for making this static? Each cpp file that includes this header will get its own text copy of this routine. We should make this non-static to avoid binary bloat.

Reply:
In order to be able to store it in `cell_order_cmp_func_`, this function needs to be static, otherwise the compiler complains.